org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/junit/JUnitFragment.xtend

```diff
@@ -100,22 +100,12 @@ class JUnitFragment extends AbstractStubGeneratingFragment {
 		].filterNull.forEach [
 			if (junitVersion == JUnitVersion.JUNIT_4) {
 				importedPackages.addAll(
-					"org.junit;version=\"4.5.0\"",
```

Comment:
do we need import package and require bundle=

Reply:
matter of taste. we might be able to drop the import package. A newly created project with JU4 already adds a Required-Bundle for org.junit 4.12

The Import-Package is added later by the JUnitFragment. Seems redundant to me.

<img width="1079" alt="screenshot 104" src="https://user-images.githubusercontent.com/265597/40139900-9dc1eb52-5951-11e8-8d06-efa09c092d46.png">

Reply:
yes this is that i meant

Reply:
removed that now. the wizard already creates test bundles with JU require bundle dependency, so the JUnitFragment does not need to do this also.

Reply:
Ok